packages/babel-plugin/src/css-prop/__tests__/index.test.tsx

```diff
@@ -122,6 +122,75 @@ describe('css prop', () => {
     );
   });
 
+  it('should evaluate simple expressions', () => {
+    const actual = transform(`
+      import '@compiled/core';
+      import React from 'react';
+
+      <div css={{ fontSize: 8 * 2 }}>hello world</div>
+    `);
+
+    expect(actual).toInclude('.cc-hash-test{font-size:16px}');
+  });
+
+  it('should bail out evaluating expression referencing a mutable identifier', () => {
```

Comment:
Do we have test like this somewhere where there is no constant violation 🤔?

```typescript
it('test statement', () => {
    const actual = transform(`
      import '@compiled/core';
      import React from 'react';

      let notMutatedAgain = 20;

      <div css={{ fontSize: notMutatedAgain }}>hello world</div>
    `);

    expect(actual).toInclude('.cc-hash-test{font-size:20px}');
  });
```

Reply:
not anymore - i'll add one

Reply:
done